lib/commands/release/index.ts

```diff
@@ -0,0 +1,108 @@
+/**
+ * @license
+ * Copyright 2016-2020 Balena Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { flags } from '@oclif/command';
+import { Release } from 'balena-sdk';
+import Command from '../../command';
+import * as cf from '../../utils/common-flags';
+import { getBalenaSdk, getVisuals, stripIndent } from '../../utils/lazy';
+
+interface FlagsDef {
+	help: void;
+	composition?: boolean;
+}
+
+interface ArgsDef {
+	commit?: string;
+}
+
+export default class ReleaseCmd extends Command {
+	public static description = stripIndent`
+		Get info for a release.
+`;
+	public static examples = ['$ balena release commit'];
+
+	public static usage = 'release <commit>';
+
+	public static flags: flags.Input<FlagsDef> = {
+		help: cf.help,
+		composition: flags.boolean({
+			default: false,
+			char: 'c',
+			description: 'Return the release composition',
+		}),
+	};
+
+	public static args = [
+		{
+			name: 'commit',
+			description: 'the commit of the release to get information',
+		},
+	];
+
+	public static authenticated = true;
+
+	public async run() {
+		const { args: params, flags: options } = this.parse<FlagsDef, ArgsDef>(
+			ReleaseCmd,
+		);
+		if (!params.commit) {
+			const { ExpectedError } = await import('../../errors');
+			throw new ExpectedError('You must specify a release commit');
+		}
+
+		const balena = getBalenaSdk();
+
+		const release = await balena.models.release.get(params.commit!);
```

Comment:
In the latest TS you shouldn't need the `!` in the `params.commit!` :thinking:
You can probably get rid of it now as well by just destructuring it (`const { commit } = params;`) before checking whether it has a value and then using the `commit` directly everywhere else.

Let's also only fetch the fields that we are interested at:
```suggestion
		const release = await balena.models.release.get(params.commit!, { $select: fields }); // the fields need to be defined top level or in a static property for this to work
```
PS: I guess you will need to switch the selected fields, depending on whether the user provided that flag or not.